tests/Controller/MediaAdminControllerTest.php

```diff
@@ -128,6 +128,10 @@ public function testListAction(): void
         $this->configureSetFormTheme($formView, ['filterTheme']);
         $this->configureSetCsrfToken('sonata.batch');
         $this->configureRender('templateList', 'renderResponse');
+
+        /**
+         * @psalm-suppress DeprecatedClass
```

Comment:
Can you solve the deprecation instead ? :)

Reply:
I'd leave it as it is right now, because phpunit [doesn't provide any replacement](https://github.com/sebastianbergmann/phpunit/issues/4026#issuecomment-825453794) for the `withConsecutive` method, and I'm way out of the context of this test case to thoroughly re-write the test. Any proposed workarounds, like using callbacks with match (switch if we still support 7.4) look very hideous to me.